Resources/config/hydra.xml

```diff
@@ -22,10 +28,12 @@
             <tag name="kernel.event_listener" event="kernel.response" method="onKernelResponse" />
         </service>
 
-        <service id="api.hydra.listener.request_exception" class="Dunglas\ApiBundle\Hydra\EventListener\RequestExceptionListener">
-            <argument type="service" id="api.hydra.normalizer.error" />
+        <service id="api.hydra.listener.exception" class="Symfony\Component\HttpKernel\EventListener\ExceptionListener">
+            <argument>api.hydra.controller.exception:showAction</argument>
```

Comment:
My bad, I thought it was the current behavior. I missed this line: https://github.com/dunglas/DunglasApiBundle/blob/1.0/Hydra/EventListener/RequestExceptionListener.php#L62

I will fix it.